### ISSUE TITLE
readme addition suggestion: docker compose

### ISSUE THREAD

**Issue by chrwei** [psybers/actual-helpers#11]
I've worked out how to run these using a docker compose instead of building a container with a Dockerfile.  This allows adding and editing scripts and .env settings without having to do a full build of the container every time so it's a much faster process when changes need made.

create a directory for the project and add this file:
docker-compose.yaml
```
services:
    nodejs:
        container_name: actual-helper
        restart: always
        image: node:22
        user: node
        command: tail -f /dev/null
        environment:
            NODE_ENV: production
        volumes:
            - ./actual-helpers:/home/node/actual-helpers:rw
        working_dir: /home/node/actual-helpers
```
git clone actual-helpers into this directory

`docker compose up -d` to start it

After starting the first time, run the command `docker exec -t actual-helper bash -c 'npm install && npm update' `.  The node_modules directory will persist.

**Comment by psybers** [comment]
I added a different docker compose file that uses the hosted Docker image for this repo.

I'll update the readme to explain how to use it, but it is similar to yours above.

**Comment by psybers** [comment]
Fixed in 33d37458e9cb62f8a61c385e10a17c89b2c835ae.
